CHANGELOG.md

```diff
@@ -82,6 +82,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed #3661 - Anonymous Proxy ipv4 whitelist does not work
 - Fixed #1847 - Delivery Service with SSL keys are no longer allowed to be updated when the fields changed are relevant to the SSL Keys validity.
 - Fixed #5153 - Right click context menu on new ag-grid tables appearing at the wrong place after scrolling. [Related github issue](https://github.com/apache/trafficcontrol/issues/5153)
+- Fixed an issue where the jobs and servers table in Traffic Portal would not clear a columns filter when its hidden
```

Comment:
because the 5.0.x branch was cut, I _think_ this belongs in the `Unreleased` section which should be very tiny at this point...

But, I don't see an unreleased section yet: https://github.com/apache/trafficcontrol/blob/master/CHANGELOG.md

Reply:
you might need to rebase this to master and when you do there will no longer be an "unreleased" section but a 5.0.0 section instead so can you create an Unreleased section and put your changelog in it?